lib/generators/react_on_rails/setup_files_generator.rb

```diff
@@ -0,0 +1,26 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class SetupFilesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def update_git_ignore
+        append_to_file ".gitignore" do
+          "# React on Rails\nnpm-debug.log\nnode_modules"
```

Comment:
Good questions I think we should add the linting gems for now. It's easy enough to comment out. We can add flags later to configure this.